File: 014_sum_two_lines/sum.py
#!/usr/bin/env python

import sys

line1 = sys.stdin.readline()
line2 = sys.stdin.readline()

numbers1 = [int(num) for num in line1.rstrip().split(',')]
numbers2 = [int(num) for num in line2.rstrip().split(',')]

sum = []
for num in xrange(0, len(numbers1)):
   sum.append(numbers1[num] + numbers2[num])

print ",".join([str(num) for num in sum])

File: 014_sum_two_lines/sum.pl
#!/usr/bin/env perl

use common::sense;

my $str1 = <>;
my $str2 = <>;

my @numbers1 = split ',', $str1;
my @numbers2 = split ',', $str2;

my @sum;
for (0 .. $#numbers1) {

    push @sum, ($numbers1[$_] + $numbers2[$_]);
}

say join ',', @sum;
